generative/maisi/maisi_diff_unet_training_tutorial.ipynb

```diff
@@ -22,15 +22,21 @@
    "id": "777b7dcb",
    "metadata": {},
    "source": [
-    "# Training a 3D Diffusion Model for Generating 3D Images with Various Sizes and Spacings"
+    "# Training a 3D Diffusion Model for Generating 3D Images with Various Sizes and Spacings\n",
+    "\n",
+    "![Generated image examples](https://developer-blogs.nvidia.com/wp-content/uploads/2024/06/image3.png)\n",
+    "\n",
+    "In this notebook, we detail the procedure for training a 3D latent diffusion model to generate high-dimensional 3D medical images. Due to the potential for out-of-memory issues on most GPUs when generating large images (e.g., those with dimensions of 512 x 512 x 512 or greater), we have structured the training process into two primary steps: 1) generating image embeddings and 2) training 3D latent diffusion models. The subsequent sections will demonstrate the entire process using a simulated dataset."
    ]
   },
   {
    "cell_type": "markdown",
    "id": "c9ecfb90",
    "metadata": {},
    "source": [
-    "## Setup environment"
+    "## Step 0: Setup environment\n",
```

Comment:
Hi @dongyang0122 , based on the the [contributing guideline](https://github.com/Project-MONAI/tutorials/blob/main/CONTRIBUTING.md), can we revert the 
```
## Step 0: Setup environment\n
```
Back to 
```
## Setup environment
```

A uniform style and format helped us to run automated scripts.

Reply:
Thanks. @mingxin-zheng I made according changes. Please proceed.

Reply:
Thanks for the edits @dongyang0122 !